apps/mochi-web/pages/overview/index.tsx

```diff
@@ -0,0 +1,37 @@
+import { Typography } from '@mochi-ui/core'
+import { ArrowRightLine, DocumentOneSolid } from '@mochi-ui/icons'
+import AuthLayout from '~components/auth-layout'
+import { NextPageWithLayout } from '~pages/_app'
+
+const App: NextPageWithLayout = () => {
+  return (
+    <AuthLayout className="bg-background-level1">
+      <div className="grid grid-cols-1 gap-4 sm:grid-cols-2">
+        <div>profile widget</div>
+      </div>
+      <div className="flex items-center p-4 mt-6 space-x-4 border rounded-lg border-divider bg-background-surface">
+        <DocumentOneSolid className="w-6 h-6" />
+        <Typography level="h8" className="flex-1">
+          Transactions
+        </Typography>
+        <ArrowRightLine className="w-6 h-6" />
+      </div>
+      <div className="mt-6">
+        <div className="py-2">
+          <Typography level="p4" className="font-medium">
+            Pay Me
+          </Typography>
+        </div>
+      </div>
+      <div className="mt-6">
+        <div className="py-2">
+          <Typography level="p4" className="font-medium">
+            Pay Link
+          </Typography>
+        </div>
+      </div>
```

Comment:
Can you break the 4 tickets
- Profile widget
- Transaction section
- Payme section
- ...

into four components/folder groups in /src/components/{PAGE}?